templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,36 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
-
-    {% if manufacturer_list %}
-      <table class="table">
+  <h1>Manufacturer List</h1>
+  
+  <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary mb-2">Create New Manufacturer</a>
+  
+  {% if object_list %}
+    <table class="table">
+      <thead>
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Actions</th> <!-- Column for update/delete actions -->
+      </tr>
+      </thead>
+      <tbody>
+      {% for manufacturer in object_list %}
         <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
+          <td>{{ manufacturer.id }}</td>
+          <td>{{ manufacturer.name }}</td>
+          <td>{{ manufacturer.country }}</td>
+          <td>
+            <a href="{% url 'taxi:manufacturer-update' manufacturer.pk %}" class="btn btn-warning btn-sm">Update</a>
+            <a href="{% url 'taxi:manufacturer-delete' manufacturer.pk %}" class="btn btn-danger btn-sm">Delete</a>
```

Comment:
Ensure that the user has the necessary permissions to update or delete a manufacturer. Consider adding permission checks if they are not already implemented.